music/src/piano_genie/test_data/README.md

```diff
@@ -0,0 +1,5 @@
+To properly test Piano Genie:
```

Comment:
remove "properly", you don't want people to test this improperly 😅

Reply:
I think we _do_ want people to be able to test this improperly. None of the other models in Magenta JS have tests which check for numeric output consistency (sidenote: I believe this is a huge mistake). I don't think we should force users to download large model checkpoints from external servers to pass the tests.